OpenRA.Mods.Common/Traits/Minelayer.cs

```diff
@@ -63,6 +63,12 @@ public class MinelayerInfo : TraitInfo, Requires<RearmableInfo>
 		[Desc("Ammo the minelayer consumes per mine.")]
 		public readonly int AmmoUsage = 1;
 
+		[Desc("Number of ticks it takes to lay a mine. The wait can be interrupted by a player order (and thus the minelaying order is canceled).")]
```

Comment:
```suggestion
		[Desc("Number of ticks it takes to lay a mine.")]
```

I really don't think that this needs to be explained. It's a basic concept in the openra engine